fx-src/index.js

```diff
@@ -24,6 +24,10 @@ import FirefoxAdapter from "./FirefoxStorage";
 export default function loadKinto() {
   const { EventEmitter } = Cu.import("resource://devtools/shared/event-emitter.js", {});
 
+  // XXX import the standalone kinto-client module landed in FFx
+  // Lets assume this import exposes a KintoHttpClient symbol:
+  Cu.import("resource://gre/modules/kinto-http-client.jsm");
```

Comment:
Yeah that was an example so you get the picture. You think this will be the final location eventually?